src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -149,6 +149,41 @@ public PersistentDataStore() {
     return messages;
   }
 
+  /**
+   * Loads Activity objects from the Datastore service that were created before the provided datetime,
+   * up to the provided limit of number of entries and sorted in descending order by creation time.
+   *
+   * @param startDatetime The retrieved activities would have been created exclusively before the startDatetime
+   * @param limit         The max. number of entries to retrieve
+   * @return A list of activities that met the criteria and sorted by creation time
+   * @throws PersistentDataStoreException if an error was detected during the load from the
+   *                                      Datastore service
+   */
```

Comment:
To improve efficiency, this loader does not follow the same logic as other loader methods existing in the PersistentDataStore, in the sense that it does not load all activities into memory. This loader only loads activities before a given Datetime, and up to a given number of entries. This would be an ideal fit for paging activities on the Activity Feed page, through a REST API.